packages/docusaurus-types/src/index.d.ts

```diff
@@ -601,11 +601,14 @@ export type TOCItem = {
 };
 
 export type ClientModule = {
+  onRouteDidUpdate?: (args: {
```

Comment:
Mmm, naming is hard. Between `onRouteUpdate`, `onRouteDidUpdate`, `onRouteWillUpdate`, I chose the former two because the third is slightly misleading: when the callback fires, the route update has already happened, it's just that the DOM hasn't re-painted yet.

Reply:
I see, that makes sense 👍